### ISSUE TITLE
Option to hide very old browsers

### ISSUE THREAD

**Issue by thany** [Fyrd/caniuse#2854]
These days, it's not always very useful to show very old versions of browsers. For example, I'm seeing Firefox versions 2.0 through today's version. Looking at projects I work on, I can imagine knowing about the past 4 versions or so, or back to the latest LTS release, would be useful. But all the way back 12 years to 2.0 doesn't make a whole lot of sense for an auto-updating browser.

The same goes for Chrome, which also auto-updates by default, so starting the list at Chrome 10 isn't too useful in most situations, I imagine.

Similar story for Safari, IE and Opera. Although IE has been deprecated entirely in favour of Edge.

Chrome for Android is strangely the reverse of this problem: it only ever shows a fairly recent version (not the current version though) but never any previous versions. Like I said, _some_ previous versions might be useful.

TLDR:
Perhaps in the Setting pane, add an option "Show deprecated" to show/hide old Firefox/Chrome/Safari/Opera/IE versions.

**Comment by cvrebert** [comment]
> Chrome for Android is strangely the reverse of this problem: it only ever shows a fairly recent version (not the current version though) but never any previous versions.

That's because caniuse currently only keeps data for the close-to-current version. See #2413.

**Comment by Fyrd** [comment]
This is why the versions that are shown (by default) are only browsers with a given usage percentage. You can change the threshold from the slider to see more or less. Between that for the default versions shown and the "Show all" button to see everything it seems adding yet another option seems excessive...

**Comment by jimmywarting** [comment]
It didn't bother me so much before but when ie10 didn't show up and chrome and firefox have released so many version that it won't fit on one screen anymore, then that is when i raise my voice.

![skarmavbild 2017-06-08 kl 10 24 34](https://user-images.githubusercontent.com/1148376/26919341-a711de12-4c35-11e7-8afb-1bb6120fc98f.png)
![skarmavbild 2017-06-08 kl 10 24 48](https://user-images.githubusercontent.com/1148376/26919347-ab358228-4c35-11e7-8c27-18b22c062032.png)

Maybe we should take a similar approach as MDN? 
Instead of showing all versions we could show "supported since x"

**Comment by Fyrd** [comment]
I believe a solution for this is now available, just requires you to set up a browser set where you can choose how far back you want browser versions to go. See https://caniuse.com/feed/159 for details.
